polling_stations/apps/file_uploads/utils.py

```diff
@@ -27,6 +27,10 @@ def assign_approved_custom_domains(user, email_domain):
             UserCouncils.objects.using(LOGGER_DB_NAME).update_or_create(
                 user=user, council=council
             )
+            # remove councils that are no longer in the custom list
+            UserCouncils.objects.using(LOGGER_DB_NAME).filter(user=user).exclude(
+                council__council_id__in=email_domain
+            ).delete()
```

Comment:
Remove 30-33 as it won't be triggered unless line 24 returns true.

Add to comment to remove the UserCouncil when removing an entry from the CUSTOM_DOMAINS